vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -456,7 +459,8 @@ void learn_or_predict(ccb& data, multi_learner& base, multi_ex& examples)
       else
       {
         // the cb example contains no action => cannot decide
-        decision_scores.push_back(data.action_score_pool.get_object());
+        decision_scores.push_back(v_array<ACTION_SCORE::action_score>());
+        data.action_score_pool.acquire_object(*decision_scores.end());
```

Comment:
`end()` is not a valid destination

Reply:
Good point. Since we have just pushed back then end -1 should be valid